pkgs/development/python-modules/click-datetime/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub
+, click }:
+
+buildPythonPackage rec {
+  pname = "click-datetime";
+  version = "0.2";
```

Comment:
```suggestion
  version = "0.2.0";
```

Reply:
This is a bit of a mess. The version up on pypi is 0.2, the git tag is 0.2.0. So I went with the pypi version instead since that is what other packages would reference.

https://pypi.org/project/click-datetime/

Reply:
pip treats 0.2.0 and 0.2 as the same version. Additionally, pip version check will look at the version written as part package installation.

Reply:
not to mention, nixpkgs-update is able update the package, if the version gets parameterized

Reply:
Updated.